.eslintrc.js

```diff
@@ -10,6 +10,8 @@ module.exports = {
     "benchmark/vendor/",
     "browser/",
     "node_modules/",
-    "*.min.js"
+    "*.min.js",
+    "build",
+    "rollup.config.js" // in .eslintrc-modules.js
```

Comment:
If we start using more modules, we can add '*.mjs' here, and use the second call to eslint to get those also.

Reply:
no, mjs doesn't exist